projects/transformers/run.py

```diff
@@ -105,7 +106,7 @@ def main():
 
         # Initialize wandb now to include the logs that follow.
         # For now, only support early wandb logging when running one experiment.
-        if len(cmd_args.experiments) == 1:
+        if is_wandb_available() and len(cmd_args.experiments) == 1:
```

Comment:
This looks good, does it fix the WAND_DISABLED=true issue as well?

Reply:
Yes, it does. I was able to reproduce that bug and fix it by editing the wandb callback class.

Reply:
Isn't is_wandb_available() still gonna check for WANDB_DISABLED and return False, hence throwing the same error?

Reply:
That error was thrown by an assert, whereas this is an if-statement. Thus, we won't get to that assert if this if-statement isn't passed first.